releasenotes/README.md

```diff
@@ -12,20 +12,29 @@ If you use [autocomplete](https://developer.salesforce.com/docs/atlas.en-us.sfdx
 
 [Click here for the v49 release notes.](./v49.md)
 
-## November 12, 2020 
+## November 19, 2020
 
 These changes are in the release candidate plug-in (`salesforcedx@latest-rc`). We plan to include these changes in next week's official release. This list isn't final and is subject to change. 
 
+* FIX: The `force:source:push|pull|status|deploy|retrieve` commands work with duplicate rules. Previously you'd get the message `No results found` and the rule wouldn't be recognized by the `source` commands.    (GitHub issue [#685](https://github.com/forcedotcom/cli/issues/685))
+
+## 50.5.0 (November 12, 2020) - CLI 7.80.0
+
+* CHANGE: The `force:apex:log:list` command displays results in descending order. Previously it displayed results in ascending order. This change results from [Salesforce Extensions for VS Code GitHub issue #2698](https://github.com/forcedotcom/salesforcedx-vscode/issues/2698), which was fixed by changing the underlying CLI command rather than VS Code. 
```

Comment:
Other than posting the github issue - I feel like last bit regarding underlying CLI command is too much info. The VS commands just launch the CLI commands under the hood. I personally would just stop (period) after github issue link.

Reply:
yeah, good point. I'll remove it. Thx.